router/core/transport.go

```diff
@@ -84,68 +96,159 @@ func (ct *CustomTransport) RoundTrip(req *http.Request) (*http.Response, error)
 	return resp, err
 }
 
+func isRunningInsideDocker() bool {
+	// Check if we are running inside docker by
+	// testing by checking if /.dockerenv exists
+	st, err := os.Stat("/.dockerenv")
```

Comment:
Could you reference and comment the source that says that's the way to identify docker?